### PR TITLE
[Technical] Tests for header and footer in DynamicTableViewController

### PR DESCRIPTION
**Pull request by dasdom**
- Test that the header/footer title is the header/footer text for enum type `.text(_:)`
- Test that header/footer title is nil if section is hidden
- Test that header/footer title is nil if it's not text.

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/__tests__/DynamicTableViewControllerTests.swift [hunk 1]
```diff
@@ -161,16 +161,141 @@ extension DynamicTableViewControllerTests {
 	
 	func testNumberOfRows_returnsValueFromViewModel_withThreeCells() {
 		// setup view model
-		let cells = [
-			DynamicCell.bold(text: "Foo"),
-			DynamicCell.bold(text: "Bar"),
-			DynamicCell.bold(text: "Baz")
+		let cells: [DynamicCell] = [
```
Comment (ChristianKienle) [inline src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/__tests__/DynamicTableViewControllerTests.swift:164]: ❤️

**Comment by ChristianKienle** [comment]
You really fell in love with this thing :D

**Review by ChristianKienle — APPROVED** [review]
Thanks again Dominik!

**Comment by dasdom** [comment]
I love to write tests. 🤷‍♀️

File: src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/__tests__/DynamicTableViewControllerTests.swift (hunk 1 above)
Comment (dasdom) [reply src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/__tests__/DynamicTableViewControllerTests.swift:164]: 🤓

**Comment by ChristianKienle** [comment]
> I love to write tests. 🤷‍♀️

You will be the perfect candidate to write tests for the new home screen. 😀 boom shakalaka

**Pull request merged**